.gitignore

```diff
@@ -3,7 +3,7 @@
 imgui.ini
 cmake-*
 ImportExecutables-*.cmake
-out
+/out*
```

Comment:
sorry, I meant `out/*` -- what I mean is that we shouldn't ignore for instance a file that would be called `outrun.txt`; it seems like `out*` would do that.
I'm curious what `out` didn't work.

Reply:
I think `out/*` would ignore all the files in the directory named out.

I wanted to have multiple build directories, out and out2 for example, and have both be ignored. 

with the /out* rule it would ignore everything starting by out, but only in the top level directory, right?

Reply:
ah I see. Okay then.